content/api/vision/coordinates.md

```diff
@@ -16,3 +17,36 @@ The Cartesian position is represented as 3 coordinates, `x`, `y` and `z`. These
 - `z` - the z coordinate, the distance forwards in front of the camera.
 
 For example, if `x`, `y` and `z` were `0`, `0` and `1` respectively, the marker would be one metre directly in front of the camera.
+
+{{% notice warning %}}
+`z` is not the same as the [spherical](#spherical-coordinates) `distance_metres`. For example, if `x`, `y` and `z` were `1`, `0` and `1` respectively, the marker would be one metre in front, and one metre to the right of the camera, and so would be `1.412` metres from the camera.
+{{% /notice %}}
+
+## Spherical Coordinates
+The spherical coordinates returned by our API differ slightly from traditional
+spherical coordinates as it is expected that the majority of markers will be at
+approximately the same level as the camera.
+
+ - `distance_metres` - the distance from the camera to the marker.
+ - `rot_x_radians` / `rot_x_degrees` - the angle of rotation in radians/degrees
+   about the Cartesian x axis.
+   \
+   An angle of 0 indicates that the marker is directly in front of the camera, a
+   positive angle indicates that the marker is in the upper part of the camera's
+   view.
+ - `rot_y_radians` / `rot_y_degrees` - the angle of rotation in radians/degrees
+   about the Cartesian y axis.
+   \
+   An angle of 0 indicates that the marker is directly in front of the camera, a
+   positive angle indicates that the marker is to the right of the camera's
+   view.
+
+{{% notice info %}}
+For example, if `distance_metres` was `1`, `rot_x_degrees` was `0`, and
+`rot_y_degrees` was `45`, the marker would be 45 degrees to the right of the
+robot, exactly 1 metre away.
+{{% /notice %}}
+
+## Diagram
```

Comment:
Nitpick, as the caption for images isnt shown on the site (working on it), can you make this an H3 heading, so it's obviously part of the previous content

Reply:
It's _not_ part of the previous content -- it's a diagram which applies to both coordinate systems.

The alt text _shouldn't appear on the site by default -- it's only _alt_ text. Are you suggesting that we need an introductory comment for the image?

Reply:
i forgot that text is alt text rather than a caption, that's fine then. I do think a comment beforehand would be good to explain what it's a diagram of, as it clearly got me confused!